bundles/org.eclipse.swt/Eclipse SWT/common/org/eclipse/swt/internal/image/SVGRasterizer.java

```diff
@@ -31,5 +32,20 @@ public interface SVGRasterizer {
 	 * @return the {@link ImageData} for the rasterized image, or {@code null} if
 	 *         the input is not a valid SVG file or cannot be processed.
 	 */
-	public ImageData rasterizeSVG(InputStream stream, int zoom) throws IOException;
+	public ImageData rasterizeSVG(InputStream stream, int zoom);
+
+	/**
+	 * Rasterizes an SVG image from the provided {@code InputStream} into a
+	 * raster image of the specified width and height.
+	 *
+	 * If {@code width} or {@code height}
+	 * is zero or negative, this method throws an {@link SWT#ERROR_INVALID_ARGUMENT}.
```

Comment:
You could document this in a `@exception` documentation for `SWTException` just like for other SWT APIs (such as the `Control` constructor to only mention one example). You could also add the explanation for `SWT#ERROR_INVALID_IMAGE` there.